.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Hey @andyatmiami! This is only a mockup gha, it prints a `Eyo, World` for now :rofl: It used to work on the  logic of the `kubeflow release`.

Reply:
So, fwiw... and I'm happy to change what I did with the `create release` PoC I did.. but I was condititioning the release creation logic on the push of a tag matching a specific pattern...

So it wouldn't need to be explicitly invoked here.. but if that "trigger of tag" won't work for reasons I am not appreciating - I am happy to adapt!